core/citrus-base/src/main/java/org/citrusframework/report/LoggingReporter.java

```diff
@@ -94,6 +94,20 @@ private static String formatDurationString(TestCase test) {
         return nonNull(test.getTestResult()) && nonNull(test.getTestResult().getDuration()) ? " (" + test.getTestResult().getDuration().toString() + ") " : "";
     }
 
+    private final boolean printStackTraces;
```

Comment:
Instead of initializing this via constructor I'd suggest to use CitrusSettings and read the default value with envVar support. This way users that are not using Spring are also able to set this via envVar or system properties.